ColorPickerPOC/ColorPickerSlider.js

```diff
@@ -0,0 +1,99 @@
+import {Cell, Row} from '@enact/ui/Layout';
+import PropTypes from 'prop-types';
+import {useCallback} from 'react';
+
+import Slider from '../Slider';
+
+import {generateOppositeColor, hexToRGB, rgbObjectToHex} from './utils';
+
+import componentCss from './ColorPickerSlider.module.less';
+
+const ColorPickerSlider = ({selectedColor, selectedColorHandler, ...props}) => {
+	const {red, green, blue} = hexToRGB(selectedColor);
+
+	const changeValueRed = useCallback((ev) => {
+		selectedColorHandler(rgbObjectToHex({red: ev.value, green, blue}));
+	}, [blue, green, selectedColorHandler]);
+
+	const changeValueGreen = useCallback((ev) => {
+		selectedColorHandler(rgbObjectToHex({red, green: ev.value, blue}));
+	}, [blue, red, selectedColorHandler]);
+
+	const changeValueBlue = useCallback((ev) => {
+		selectedColorHandler(rgbObjectToHex({red, green, blue: ev.value}));
+	}, [green, red, selectedColorHandler]);
+
+	return (
+		<div {...props} style={{height: '320px', width: '450px'}}>
```

Comment:
let's move height and width to a classname in less file. probably you will have to double the values to respect 4k resolution

Reply:
Done